### PR TITLE
[CEP 012] Roadmap for Public OCaml Interfaces.

### PR DESCRIPTION
**Pull request by ejgallego**

### CODE REVIEW

File: text/012-coq-ml-interfaces.md [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+```
+
+Then, we can define our old types using regular recursive types:
+```ocaml
+type parse_expr = (pvar, pcase, psyntax, parse_expr) coq_expr
+type resolved_expr = (global_reference, pcase, psyntax, parse_expr) coq_expr
+```
+and our transformations:
+```ocaml
+resolve_refs : parse_expr -> resolve_refs
```
Comment (SkySkimmer) [inline text/012-coq-ml-interfaces.md]: Presumably `-> resolved_expr`

Comment (ejgallego) [inline text/012-coq-ml-interfaces.md]: Indeed, thanks a lot Gaetan!

**Comment by gares** [comment]
@ejgallego I think we should re discuss this CEP business at a WG, this one was kind of lost in the mist.
Can you write an update (since many things are already implemented and merged)? What is jet to be done?

**Comment by ejgallego** [comment]
Indeed a few parts are done, others not, will try to update.

**Pull request closed without merging**